specs/opcode/F3RETURN.md

```diff
@@ -43,6 +43,7 @@ the following parameters:
     - `dst_addr: 0`
     - `rw_counter: callee.rw_counter`
     - `rwc_inc: length`
+
 D. Otherwise, if it's not a root call, it copies the source memory chunk to the
```

Comment:
Can we also fix the order of the A, B, C, and D here? We can remove them all if you like.

Reply:
I wrote them in this order to follow an `if else` logic.  It's something like: (if X then A else D) and (if Y then B else C).
I used letters instead of numbers because I thought having unsorted numbers would be more confusing.

Reply:
Oh, I see. Then LGTM now.